multiplayer/src/index.css

```diff
@@ -6,6 +6,6 @@
     --slot-0-color: 77, 77, 77;
     --slot-1-color: 255, 0, 0;
     --slot-2-color: 0, 56, 255;
-    --slot-3-color: 77, 166, 77;
-    --slot-4-color: 228, 204, 0;
+    --slot-3-color: 255, 154, 20;
+    --slot-4-color: 77, 166, 77;
```

Comment:
Intentional to change slot 4 as well?

Reply:
Yes, orange (previously yellow) is play 3 color and green is p4, so swapping green /orangeyellow